src/components/IntlTelInputApp.js

```diff
@@ -719,7 +719,7 @@ class IntlTelInputApp extends Component {
       // invalid dial code, so empty
       // Note: use getNumeric here because the number has not been
       // formatted yet, so could contain bad chars
-      countryCode = '';
+      countryCode = null;
```

Comment:
By providing null instead, line 728 wont run, causing the flag to not update to an invalid value.